Packs/SAP_BTP/Integrations/SAPBTP/SAPBTP.yml

```diff
@@ -0,0 +1,180 @@
+- Connect
+- Collect
+configuration:
+- name: url
+  display: API URL (Audit Log Service)
+  required: true
+  type: 0
+  section: Connect
+  additionalinfo: SAP Service Key field 'url' for Audit Log API (e.g., https://auditlog-management.cfapps.<region>.hana.ondemand.com).
+- name: token_url
+  display: Token URL (Authentication Service)
+  required: true
+  type: 0
+  section: Connect
+  additionalinfo: For Non-mTLS use 'uaa.url' (e.g., https://<subdomain>.authentication.<region>.hana.ondemand.com). For mTLS use 'uaa.certurl' (e.g., https://<subdomain>.authentication.cert.<region>.hana.ondemand.com).
+- name: client_id
+  display: Client ID
+  required: true
+  type: 0
+  section: Connect
+  additionalinfo: SAP Service Key field 'uaa.clientid'. Required for both mTLS and Non-mTLS authentication.
+- name: auth_type
+  display: Authentication Type
+  required: true
+  type: 15
+  section: Connect
+  defaultvalue: Non-mTLS
+  options:
+  - mTLS
+  - Non-mTLS
+  additionalinfo: Select authentication method. mTLS (recommended) requires Certificate + Private Key. Non-mTLS requires Client Secret.
+- name: client_secret
+  display: Client Secret
+  displaypassword: Client Secret
+  required: false
+  type: 9
+  section: Connect
+  hiddenusername: true
+  additionalinfo: For Non-mTLS authentication. SAP Service Key field 'uaa.clientsecret'.
+- name: certificate
+  display: Certificate
+  required: false
+  type: 12
+  section: Connect
+  additionalinfo: For mTLS authentication (recommended). Body of certificate.pem file in PEM format.
+- name: private_key
+  display: Private Key
+  required: false
+  type: 14
+  section: Connect
+  additionalinfo: For mTLS authentication (recommended). Body of key.pem file in PEM format.
+- name: insecure
+  display: Trust any certificate (not secure)
+  required: false
+  type: 8
+  defaultvalue: "false"
+  section: Connect
+  advanced: true
+- name: proxy
+  display: Use system proxy settings
+  required: false
+  defaultvalue: "false"
+  type: 8
+  section: Connect
+  advanced: true
+- name: first_fetch
+  display: First fetch time
+  required: false
+  type: 0
+  section: Collect
+  defaultvalue: ""
+  additionalinfo: >-
+      Time range to fetch events from on first run. Leave empty to start from current time (recommended).
+      Only use this to fetch historical data if specifically needed (e.g., "3 days", "1 hour").
+- name: incidentType
+  display: Incident type
+  required: false
+  type: 13
+  section: Collect
+- name: max_fetch
+  display: The maximum number of audit logs per fetch
+  required: false
+  type: 0
+  section: Collect
+  advanced: true
+  defaultvalue: "5000"
+  additionalinfo: Maximum number of events to fetch per cycle (default 5000).
+script:
+  commands:
+  - name: sap-btp-get-events
+    description: |-
+      Gets events from SAP BTP Audit Log. This command is used for developing/debugging and is to be used with caution, as it can create events, leading to event duplication.
```

Comment:
```suggestion
      Gets events from SAP BTP Audit Log. This command is used for developing/ debugging and is to be used with caution, as it can create events, leading to events duplication and API request limitation exceeding.
```